backend/coreapp/tests.py

```diff
@@ -683,15 +684,31 @@ def create_test_project(self, slug: str = "test") -> Project:
 
         return project
 
-    """
-    Test that the repo is cloned into a directory and that it is deleted when the db object is deleted.
-    """
-    def test_create_delete_repo_dir(self):
+    @patch("coreapp.github.subprocess")
+    @patch("pathlib.Path.mkdir")
+    def test_create_repo_dir(self, mock_mkdir, mock_subprocess):
+        """
+        Test that the repo is cloned into a directory
+        """
         project = self.create_test_project()
-
         project.repo.pull()
-        self.assertTrue(project.repo.get_dir().is_dir())
 
+        mock_subprocess.assert_called_once()
+        self.assertListEqual(
+            mock_subprocess.calls[0][:2],
+            ["git", "clone", f"https://github.com/{owner}/{repo}"]
+        )
```

Comment:
I think this should look at any of the calls, not just the 0th

Reply:
There's only one call (a fact that was asserted on the previous line)

Reply:
oop, my bad :+1: